apps/frontend/src/features/psi/InteractivePSI.tsx

```diff
@@ -360,6 +391,66 @@ const InteractivePSI: React.FC<InteractivePSIProps> = ({
     }
   };
 
+  const getOverlapDisplay = () => {
+    if (!overlap) return null;
+
+    return (
+      <div className="flex flex-col gap-4">
+        {overlap?.sharedConnections && overlap.sharedConnections.length > 0 && (
+          <div>
+            <h3 className="font-medium text-bold text-primary text-sm font-sans mb-2">
+              Shared Connections
+            </h3>
+            <ul className="list-disc list-inside">
+              {overlap.sharedConnections.map((user, index) => (
+                <li key={index} className="text-tertiary text-sm">
+                  {user}
```

Comment:
Consider linking to their contact? As soon as profile pics are available it'd be great to use.

Reply:
added